src/repo/__tests__/github.js

```diff
@@ -65,6 +65,10 @@ test('handle errors', async () => {
   await rejects(getUserInfo('nodisplayname'))
 })
 
+test('Throw error when no username is provided', () => {
+    expect(getUserInfo).toThrow('No login when adding a contributor. Please specify a username.')
```

Comment:
@sinchang I did it like this. Re read documentation. Actually expect should call the function not us.

Reply:
@Berkmann18 friendly ping